pod.go

```diff
@@ -85,6 +85,18 @@ var (
 		"The number of requested memory bytes  by a container.",
 		[]string{"namespace", "pod", "container", "node"}, nil,
 	)
+
+	descPodContainerLimitedCpuCores = prometheus.NewDesc(
+		"kube_pod_container_limited_cpu_cores",
+		"The number of limited cpu cores by a container.",
```

Comment:
This is not your fault (it matches the request metrics) but I don't like the "limited X" / "requested X" wording here. @brancz WDYT about changing this before the next release? I would prefer `kube_pod_container_{limits,requests}_{cpu_cores,memory_bytes}` to match the hierarchy in the pod spec.